### ISSUE TITLE
Describe npm, node version for each release in the doc

### ISSUE THREAD

**Issue by c9s** [meteor/meteor#7514]
To avoid the package issues like "bcrypt invalid ELF", I suggest we should describe the version of npm, node used for different release.

And we can ask the users to install the correspond node, npm versions to make the local/server environment consistent.

**Comment by DerekTBrown** [comment]
1) The version history is actually published in the changelog:
https://github.com/meteor/meteor/blob/devel/History.md#v14

2) Meteor does not use system copies of node and npm- it automatically installs bundled versions of npm, node and mongodb, inside the meteor installation directory.  That means if you install Meteor 1.4 on your local machine, and on your server, the node, npm and mongodb versions are guaranteed to be the same.  You can access these copies by using: `meteor mongo`, `meteor npm` and `meteor node`.

3) This seems to be an issue with your installation of [bcrypt](http://stackoverflow.com/questions/15809611/bcrypt-invalid-elf-header-when-running-node-app).  Make sure you are installing it using `meteor npm install --save bcrypt`, and that you are not trying to ship compiled code across systems.

**Comment by DerekTBrown** [comment]
Move to close issue- not related to Meteor.

**Comment by c9s** [comment]
@DerekTBrown the bundled npm, node only runs on local side, if you deploy the bundle onto the server,  the bundled npm and node are not included, thus things like *mup* needs to install a node externally to run the app. and this is causing the issues, not just bcrypt

**Comment by c9s** [comment]
the bcrypt is not installed by ourself, it was installed by meteor.

**Comment by abernix** [comment]
It seems like you're talking about server side, and you're right – the information wasn't out there at the same time 1.4 was released.  However, within the last couple days, [the Meteor Guide section on custom deployment](http://guide.meteor.com/deployment.html#custom-deployment) was updated with this information (meteor/guide#528) and the README file included in each bundle was updated with that information thanks to (#7487).

Third-party options like Mup will need to change its practices a bit (and I've already documented that elsewhere in #7475).

Closing as I'm not sure that information belongs anywhere else.
